### PR TITLE
feat: update nixpkgs-mozilla

### PR DESCRIPTION
**Pull request by K900**

### CODE REVIEW

File: latest.json [hunk 1]
```diff
@@ -5,25 +5,29 @@
...
+        "wmClass": "firefox",
+        "channel": "release"
       },
       "firefox-esr-bin": {
         "name": "Firefox ESR",
         "release": true,
         "version": "102.15.1esr",
-        "wmClass": "firefox"
+        "wmClass": "firefox",
+        "channel": "release"
```
Comment (colemickens) [inline latest.json:23]: Just checking, these are right? no `channel: "esr"`?

Comment (K900) [reply latest.json:23]: This doesn't actually affect anything (yet), we can discuss special-casing ESR in https://github.com/NixOS/nixpkgs/pull/256292

**Comment by colemickens** [comment]
The update process will clobber this change on next update: https://github.com/nix-community/flake-firefox-nightly/blob/80a0b41ecd264c2a760fdb42a1daed6a63cab7eb/update.sh#L9-L23

I think you'd want to (handwaving) have this metadata in the flake and merge it into `latest` before it's output to json and re-saved.

**Comment by K900** [comment]
It won't because I also added it upstream.

**Comment by K900** [comment]
(force pushed to reorder the keys to match)

**Comment by colemickens** [comment]
ah, I see: https://github.com/mozilla/nixpkgs-mozilla/pull/321/files#diff-79c27ec9d167a619cbe33cdcbed00fa5962046a52662b9ff19d43266ea9027eaR159-R186

Looks good then.

**Pull request merged**